payments/utils.py

```diff
@@ -343,7 +343,12 @@ def approve_order(
             order.lease.application.status = ApplicationStatus.OFFER_SENT
             order.lease.application.save()
 
-    send_payment_notification(order, request, email, phone_number=order.customer_phone)
+    if order.customer.is_non_billable_customer():
+        order.set_status(OrderStatus.PAID, "Non-billable customer.")
```

Comment:
Question to mostly Alex: In CreateBerthLeaseMutation and CreateWinterStorageLeaseMutation we use PAID_MANUALLY for non-billable customers, here it's PAID. We should use the same status in both cases, right? I remember discussion that PAID_MANUALLY is a better fit here, and PAID should be reserved for customers that have paid through the "normal" online channel. Anyway, the handling should be uniform.

Reply:
Ahhh good point! I remember we had talked about `PAID` vs `PAID MANUALLY` somewhere else, then we should definitely go with the later. 👍

Reply:
Good point! Fixed :+1: